src/main/java/com/bmc/truesight/saas/meter/client/model/Event.java

```diff
@@ -34,20 +42,42 @@ public Instant timestamp() {
     public abstract Optional<String> at();
     public abstract Optional<String> ad();
     public abstract Optional<String> sender();
+    public abstract Optional<Map<String, String>> properties();
```

Comment:
i think using optional here is overkill. It will default to an empty map, which is fine.